File: Feitos em Java/2534.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        while(leitor.hasNext()){
            int n = leitor.nextInt();
            int q = leitor.nextInt();
            int[] vet = new int [n];
            int[] aux = new int [n];
            for(int i=0;i<n;i++){
                vet[i]=leitor.nextInt();
            }
            int aux2;
            for(int i=0;i<n;i++){
                for(int j=i+1;j<n;j++){
                    if(vet[i]<vet[j]){
                        aux2=vet[i];
                        vet[i]=vet[j];
                        vet[j]=aux2;
                    }
                }
            }
            for(int i=0;i<q;i++){
                int p = leitor.nextInt();
                System.out.println(vet[p-1]);
            }
	}
    }
}

File: Feitos em C/2534.c
#include <stdio.h>
int main() {
	int n,q,i,j;
	while(scanf("%d",&n)==1){
		scanf("%d",&q);
		int vet[n],aux[n];
		for(i=0;i<n;i++){
			scanf("%d",&vet[i]);
		}
		int aux2;
		for(i=0;i<n;i++){
			for(j=i+1;j<n;j++){
				if(vet[i]<vet[j]){
					aux2=vet[i];
					vet[i]=vet[j];
					vet[j]=aux2;
				}
			}
		}
		for(i=0;i<q;i++){
			int p;
			scanf("%d",&p);
			printf("%d\n",vet[p-1]);
		}
	}
    return 0;
}
